lib/rubocop/cop/lint/eval.rb

```diff
@@ -8,9 +8,10 @@ class Eval < Cop
         MSG = 'The use of `eval` is a serious security risk.'
 
         def on_send(node)
-          receiver, method_name, = *node
+          receiver, method_name, *args = *node
 
-          add_offense(node, :selector) if receiver.nil? && method_name == :eval
+          return unless receiver.nil? && method_name == :eval && args.first.type != :str
```

Comment:
Man, I am getting schooled here. OK, PR updated. Also added a spec for that case.